src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```

Comment:
```suggestion
    transition: color 300ms;
```

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/207929929-bcfe163c-9add-4da4-838b-d143cb3092d4.png)